src/modules/deal/dto/deals-request.dto.ts

```diff
@@ -0,0 +1,73 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsDateString, IsNotEmpty, IsNumber, IsOptional, IsString, MaxLength } from 'class-validator';
+
+export class CreateDealDto {
+  @ApiProperty({
+    example: '0020900035', // Comment on PR if we need a description when it's self evident
+  })
+  @IsString()
+  @IsNotEmpty()
+  @MaxLength(10)
```

Comment:
Please add `@Min(x)`

Reply:
What is the minimum value? 😕 I only saw a maximum value specified in the Anypoint documentation:
![image](https://user-images.githubusercontent.com/6897716/225072583-cc0e453b-f00b-45db-ad87-bd5ea7a0d3f4.png)

Reply:
Deal, Facility and Covenant IDs are 10 characters long.
Party, Customer, Obligor and similar IDs are 8 characters long

We can change validation to `@Lenght(10)` or `@Lenght(10)`. 
We also should allow just digits for this string using  @Matches(/^\d{10}$/).
We could develop custom UKEF ID validation at some point because these Ids are used multiple times.